data/process_query_results.py

```diff
@@ -3,165 +3,72 @@
-total_cpu_time_May2021['tool_name_version'] = total_cpu_time_May2021.apply(lambda x: x['tool_name_version'] + getName(str(x['tool_id'])) + "/" + getVersion(str(x['tool_id'])),axis =1)
-total_cpu_time_May2021 = total_cpu_time_May2021.drop(columns = ['tool_id'])
-total_cpu_time_May2021.to_csv('total_cpu_time_May2021.csv')
-
-## Get the tool_id in the query results for total_memory for May 2021 to be in the format tool_name/version
-totalmemory_May2021 = pd.read_csv("../unprocessed_toolid/totalmemory_May2021.txt",
-                    delimiter = '|',names = ['month_year','tool_id','totalmemory'],skiprows = 1)
-totalmemory_May2021 = totalmemory_May2021.dropna() # Removes all the Nan values
-totalmemory_May2021['tool_id'] = totalmemory_May2021['tool_id'].str.strip()
-totalmemory_May2021['tool_name_version'] = ''
-totalmemory_May2021['tool_name_version'] = totalmemory_May2021.apply(lambda x: x['tool_name_version'] + getName(str(x['tool_id'])) + "/" + getVersion(str(x['tool_id'])),axis =1)
-totalmemory_May2021 = totalmemory_May2021.drop(columns = ['tool_id'])
-totalmemory_May2021.to_csv('totalmemory_May2021.csv')
-
-## Get the tool_id in the query results for avg_cpu_time for May 2021 to be in the format tool_name/version
-avg_cpu_time_May2021 = pd.read_csv("../unprocessed_toolid/avg_cpu_time_May2021.txt",
-                    delimiter = '|',names = ['tool_id','month_year','avg_cpu_time'],skiprows = 1)
-avg_cpu_time_May2021 = avg_cpu_time_May2021.dropna() # Removes all the Nan values
-avg_cpu_time_May2021['tool_id'] = avg_cpu_time_May2021['tool_id'].str.strip()
-avg_cpu_time_May2021['tool_name_version'] = ''
-avg_cpu_time_May2021['tool_name_version'] = avg_cpu_time_May2021.apply(lambda x: x['tool_name_version'] + getName(str(x['tool_id'])) + "/" + getVersion(str(x['tool_id'])),axis =1)
-avg_cpu_time_May2021 = avg_cpu_time_May2021.drop(columns = ['tool_id'])
-avg_cpu_time_May2021.to_csv('avg_cpu_time_May2021.csv')
-
-## Get the tool_id in the query results for avgmemory for May 2021 to be in the format tool_name/version
-avgmemory_May2021 = pd.read_csv("../unprocessed_toolid/avgmemory_May2021.txt",
-                    delimiter = '|',names = ['month_year','tool_id','avgmemory'],skiprows = 1)
-avgmemory_May2021 = avgmemory_May2021.dropna() # Removes all the Nan values
-avgmemory_May2021['tool_id'] = avgmemory_May2021['tool_id'].str.strip()
-avgmemory_May2021['tool_name_version'] = ''
-avgmemory_May2021['tool_name_version'] = avgmemory_May2021.apply(lambda x: x['tool_name_version'] + getName(str(x['tool_id'])) + "/" + getVersion(str(x['tool_id'])),axis =1)
-avgmemory_May2021 = avgmemory_May2021.drop(columns = ['tool_id'])
-avgmemory_May2021.to_csv('avgmemory_May2021.csv')
-
+def normalizeToolNameVersion(dataset):
+    toolNameVersion = dataset.apply(lambda x: x['tool_name_version'] + getToolName(str(x['tool_id'])) + "/" + getVersion(str(x['tool_id'])),axis =1)
+    return toolNameVersion
+
+def normalizeHeaders(columns, isPivot):
```

Comment:
The Python convention is to using underscores to separate words instead of camel case (https://peps.python.org/pep-0008/#function-and-variable-names) so would be nice to update all the function names.

Reply:
Thanks for the comment (and link!). Updated.